presto-hive-hadoop2/conf/alluxio-docker.yml

```diff
@@ -0,0 +1,35 @@
+version: '2'
+services:
+  alluxio-master:
+    hostname: alluxio-master
+    image: '${ALLUXIO_BASE_IMAGE}:${ALLUXIO_IMAGE_TAG}'
+    command: master
+    environment:
+      ALLUXIO_MASTER_JAVA_OPTS: '
```

Comment:
Fixed. Ended up using  `>` instead because the behavior of `|` is to preserve newlines which doesn't properly pass all config values to the alluxio process.